Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/how-to-write-a-zkapp-ui.mdx

---
title: How to Write a zkApp UI
hide_title: true

> description: This guide outlines how to write a user interface for a zkApp, which includes a smart contract and a UI to interact with it. It covers

Reviewer: ```suggestion
description: Guides steps to write a user interface for a zkApp that includes a smart contract and a UI to interact with it. ```
